contracts/inx/INXCommitment.sol

```diff
@@ -0,0 +1,184 @@
+pragma solidity 0.4.24;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "openzeppelin-solidity/contracts/lifecycle/Pausable.sol";
+
+import "./INXCrowdsale.sol";
+import "./INXToken.sol";
+
+/**
+ * @title INXCommitment used to capture commitments to the INX token sale from an individual address.
+ * Once KYC approved can redeem to INX Tokens.
+ */
+contract INXCommitment is Pausable {
+    using SafeMath for uint256;
+
+    address internal sender;
+
+    uint256 internal tokenBalance;
+
+    bool internal refunding = false;
+
+    INXCrowdsale internal crowdsale;
+    INXToken internal token;
```

Comment:
needs signatures interfaces for `INXToken` & `INXCrowdsale`

Reply:
Done